install_files/ansible-base/roles/ossec_server/tasks/authd.yml

```diff
@@ -1,7 +1,8 @@
 ---
-  # The authd ossec process automatically adds the app server ossec agent it is
-  # not an authenticated process. There is an option we could add to create and
-  # verify the server/agents ssl keys with the root CA cert. see
+  # The authd ossec process automatically adds the app server ossec agent.
+  # This addition is not an authenticated process. There is an option
+  # we could add to create and  verify the server/agents ssl keys with
+  # the root CA cert, see:
   # http://ossec-docs.readthedocs.org/en/latest/programs/ossec-authd.html
```

Comment:
I agree with @dolanjs, I don't think the security benefit is worth the added overhead and complexity in this case. It makes more sense for a more open network that needs to dynamically add and remove OSSEC agents.